### PR TITLE
Shayan/Market selector issues

### PR DESCRIPTION
**Pull request by shayan-binary-2**
- Fix the issue of the market selector button
 - Clearing search after picking new market

### CODE REVIEW

File: sass/styles/chart.scss [hunk 1]
```diff
@@ -221,7 +221,7 @@ html, body {
     padding: 0px;
     margin-bottom: 17px;
     border-radius: 4px;
-    z-index: 12;
+    z-index: 15;
```
Comment (salar-fs) [inline sass/styles/chart.scss:224]: `15` is the minimum value that works? Could we use a variable with a good name for this?

Comment (shayan-binary-2) [reply sass/styles/chart.scss:224]: @salar-fs  to be honest, we don't have exact valid range for that. It comes to `15` as 12 seems to have a conflict with barrier shade. that would be a good idea to have constant numbers for them, we should handle it in refactoring

File: src/store/ChartTitleStore.js [hunk 2]
```diff
@@ -51,6 +51,7 @@ export default class ChartTitleStore {
     @observable todayChange = null;
     @observable todayChangePercent = null;
     @observable isVisible = false;
+    @observable enableShowPrice = false;
```
Comment (balakrishna-binary) [inline src/store/ChartTitleStore.js:54]: making `enableShowPrice` as observable is not required here. There is no reaction depending on this. I think we can just use it as normal variable

Comment (shayan-binary-2) [reply src/store/ChartTitleStore.js]: yes, you are right. nice catch, just updated it

**Review by balakrishna-binary — APPROVED** [review]

**Review by salar-fs — APPROVED** [review]

**Pull request merged**